### ISSUE TITLE
Translation issues

### ISSUE THREAD

**Issue by shivani2910** [SocialEngine/phpv4-issues#5]
## Bug:

Following phrases are missing in .csv files of respective plugins because of which admins of multilingual websites are unable to translate them in other languages:

**Core**

```
Accept Request
Admin Settings  
A user account with that email was not found
appears to be a local network name but local network names are not allowed
appears to be a DNS hostname but cannot extract TLD part
can not be matched against quoted-string format
Birthdays must include a month and a date
(you) , In file: In application/modules/User/controllers/FriendsController.php around line 737:
```

**Forum**

```
body    
Nobody has created a forum with that criteria
Topic Moved
Move Topic
There are no forums yet.
Forum posts
```

**Music**

```
Please enter a title or title or select a playlist.
was not found in the haystack
Recent Playlists
Popular Playlists
Only my friend's playlists  
```

**Event**

```
Search Events
everyone's events
Recent Events
```

**Invite**

```
Request Invite
```

**Group**

```
Cancel Membership Request
Popular
Request Membership
Recent Groups
Ignore Membership Request
Accept Membership Request   
```

**Classified**

```
popular classifieds
Recent Classifieds
```

**Poll**

```
Close Poll  
Open Poll   
Recent Polls
Popular Polls
```

**User**
Admin Panel:

```
History has been cleared.
Your changes have been saved.
```
Front end:
```
Save Photo
```

Some other translation issues are reported at: https://github.com/SocialEngine/phpv4-issues/issues/464
## Solution:

We will add these phrases to the .csv files of our modules in the next release. For now, if you want to translate any of these phrases, then you may go to Admin > Layout > Language Manager section of your website and add these phrases clicking on "Add new Phrase" button and then translate them and save changes.

**Comment by Tomislav20** [comment]
Hi,

Request Invite phrase should go in event module instead of invite.csv. See the screenshot I've attached.

Also in Groups there should be "Popular Groups" instead of "Popular".

Best Regards

![request invite](https://cloud.githubusercontent.com/assets/12753666/7995226/7082873e-0b14-11e5-898e-a6c14ffa0ee5.jpg)

**Comment by DonnaScriptTechs** [comment]
This is fixed for 4.9.0 when it is released.

**Comment by Elshara** [comment]
Awesome!
